src/core/runner.ts

```diff
@@ -320,6 +323,28 @@ export default class Runner {
     });
     // clear screenshots cache after each journey
     await rm(Runner.screenshotPath, { recursive: true, force: true });
+
+    const pages = driver.context.pages();
+    const videoDir = options.playwrightOptions?.recordVideo?.dir;
```

Comment:
We should not record video based on this, We should hide the internals of PW and let our runner take care of the video option with a value like `on|off|only-on-failure` like screenshots and ask users to pass that flag. Based on that, the runner would store the rest on the provided `output-dir` if present and write those video files once the test is completed and based on the video flag. 

By this way, you would just give the `path` to the reporter and the reporter can just use that path and share it to the buildkite CLI where the artifacts are present. 

Screenshots would be the same as well.